src/main/java/com/opencart/pages/SearchProductResultPage.java

```diff
@@ -13,14 +13,19 @@
 public class SearchProductResultPage extends BasePage {
     @FindBy(xpath = ".//div/h2")
     private WebElement successTitle;
-
+    @FindBy(xpath = ".//p[contains(text(),'no product')]")
```

Comment:
Still not fully correct, imagine the situation when that text will change. Try to rewrite it, using the "following-sibling" Xpath function.

Reply:
Arsen, thanks for your suggestion for the xPath. Is this one will be correct?
.//div/h2/following-sibling::p

Reply:
Yep, it would be better

Reply:
or
.//div/h2/following-sibling::*

Reply:
It is up to you, but I would recommend using this one [.//div/h2/following-sibling::p]